quokka_editor_front/src/components/molecules/addDocument/AddDocument.tsx

```diff
@@ -1,28 +1,40 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { useAppDispatch } from "../../../Redux/hooks";
 import { API_URL } from "../../../consts";
 import axios from "axios";
 import { addDocument } from "../../../Redux/documentsSlice";
 import Modal from "../../misc/Modal";
+import { TemplateType } from "../../../types/global";
+import { useNavigate } from "react-router-dom";
 
 const AddDocument: React.FC = () => {
   const dispatch = useAppDispatch();
+  const navigate = useNavigate();
   const [addModal, setAddModal] = useState(false);
-  const [title, setTitle] = useState("");
+  const [templates, setTemplates] = useState<TemplateType[]>([]);
+  const [pickedTemplate, setPickedTemplate] = useState<string>("");
+  useEffect(() => {
+    axios
+      .get(API_URL + "templates/", {
```

Comment:
Also, I think you should add urls.ts or smth to have all APP URLs in one place. Then if one of the URLs changes you will need to change it in one place. Use more consts